.pipelines/java_packaging.yml

```diff
@@ -72,11 +63,15 @@ stages:
         SourceFolder: 'out/$(Agent.OS)/RelWithDebInfo/java/build/libs'
         TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
-    - task: PublishPipelineArtifact@1
-      displayName: 'Publish Linux Artifact'
-      inputs:
-        targetPath: '$(Build.ArtifactStagingDirectory)'
-        artifact: 'drop-linux-onnxruntime-extensions-java-cpu'
+    - script: |
+        cd $(Build.ArtifactStagingDirectory)
+        jar xvf onnxruntime-extensions-0.11.0.jar
```

Comment:
need read the version number from version.txt

Reply:
good catch, thanks, done!